### DISCUSSION TITLE
mobx-react dedicated documentation

### DISCUSSION THREAD

**Discussion by akaguny** [mobxjs/mobx#2491] [Q&A] [answered]
what about deploy mobx-react for earlier version of mobx to ghpages? We really want, but don't move to mobx6 with new mobx-react immediately

**Comment by danielkcz** [comment, marked as answer]
What about the old site you are you missing in the new docs?

**Reply by akaguny** [reply]
example: useObserver example and description

**Reply by mweststrate** [reply]
https://github.com/mobxjs/mobx-react-lite#useobservertfn---t-basecomponentname--observed-options-iuseobserveroptions-t-deprecated

**Comment by danielkcz** [comment]
Just for the info, I've revived the https://mobx-react.js.org/ and included deprecation notice instead of hard redirect.

**Reply by akaguny** [reply]
thanks a lot =)
